src/EA.Weee.Web.Tests.Unit/Areas/Admin/Controllers/AddPcsControllerTests.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace EA.Weee.Web.Tests.Unit.Areas.Admin.Controllers
+{
+    public class AddPcsControllerTests
```

Comment:
Empty test class?

Reply:
Will be populated when we implement the Add pcs functionality